src/components/sdk/ContentShare/index.tsx

```diff
@@ -10,37 +10,49 @@ import { BaseSdkProps } from '../Base';
 
 interface Props extends BaseSdkProps {
   nameplate?: string;
+  tileId?: number;
 }
 
 export const ContentShare: React.FC<React.PropsWithChildren<Props>> = ({
   className,
+  tileId,
   ...rest
 }) => {
   const audioVideo = useAudioVideo();
-  const { tileId } = useContentShareState();
+  const contentShareState = useContentShareState();
+
+  // Use the provided tileId or fall back to the default (for backward compatibility)
+  const tileIdToRender =
+    tileId !== undefined ? tileId : contentShareState.tileId;
+
+  const attendeeId = tileIdToRender
+    ? contentShareState.tileIdToAttendeeId[tileIdToRender.toString()]
+    : null;
+
   const videoEl = useRef<HTMLVideoElement | null>(null);
 
   useEffect(() => {
-    if (!audioVideo || !videoEl.current || !tileId) {
+    if (!audioVideo || !videoEl.current || !tileIdToRender) {
       return;
     }
 
-    audioVideo.bindVideoElement(tileId, videoEl.current);
+    audioVideo.bindVideoElement(tileIdToRender, videoEl.current);
 
     return () => {
-      const tile = audioVideo.getVideoTile(tileId);
+      const tile = audioVideo.getVideoTile(tileIdToRender);
       if (tile) {
-        audioVideo.unbindVideoElement(tileId);
+        audioVideo.unbindVideoElement(tileIdToRender);
       }
     };
-  }, [audioVideo, tileId]);
+  }, [audioVideo, tileIdToRender]);
 
-  return tileId ? (
+  return tileIdToRender ? (
     <ContentTile
       objectFit="contain"
-      className={className || ''}
+      className={`ch-content-share--${tileIdToRender} ${className || ''}`}
       {...rest}
       ref={videoEl}
+      data-content-share-attendee={attendeeId}
```

Comment:
Is this for testing?

Reply:
yeah, this is to make it easier for builder to know which tile is for which attendee so they can locate the element quickly.